src/mugs.js

```diff
@@ -253,10 +271,18 @@ define([
          * this to drop "info" messages.
          */
         getErrors: function () {
-            return _.uniq(this.messages.get());
+            var errors = [];
+            var messagesList = this.messages.messages.nodeID || [];
+            for (var i = 0; i < messagesList.length; i++) {
+                var messageObj = messagesList[i];
+                if (messageObj.level === 'error' || messageObj.level === 'warning') {
+                    errors.push(messageObj.message);
+                }
+            }
+            return _.uniq(errors);
```

Comment:
This is dropping messages for properties other than `nodeID`. I think you want something like
```js
var INFO = this.INFO,
    errors = [];
this.messages.each(function(msg) {
    if (msg.level !== INFO) {
        errors.push(msg.message);
    }
});
return _.uniq(errors);
```